src/solvers/boolector/CMakeLists.txt

```diff
@@ -13,8 +13,7 @@ else()
     CPMAddPackage(
       NAME boolector
       DOWNLOAD_ONLY YES
-      GITHUB_REPOSITORY Boolector/boolector
-      GIT_TAG 3.2.3)
+      GITHUB_REPOSITORY rafaelsamenezes/boolector)
```

Comment:
Why does this refer to your personal repo now?

Reply:
Because Boolector cmake is old and the project is archived

Reply:
Why can't we just patch the CMake file after checking out upstream?
Alternatively, could we move this repo under the esbmc organization?

Reply:
> Why can't we just patch the CMake file after checking out upstream?

That's definitely a better idea.